apps/account-api/test/delegations.controller.e2e-spec.ts

```diff
@@ -119,6 +120,58 @@ describe('Delegation Controller', () => {
   describe('/v1/delegation/revokeDelegation', () => {
     let delegationObj: RevokeDelegationPayloadResponseDto;
 
+    it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with invalid accountId', async () => {
+      const providerId = provider.msaId?.toString();
+      const { keypair } = users[1];
+      const invalidAccountId = `${keypair.address.slice(0, -1)}5`;
+      const getPath: string = `/v1/delegation/revokeDelegation/${invalidAccountId}/${providerId}`;
+      await request(httpServer).get(getPath).expect(400).expect({
+        statusCode: 400,
+        message: 'Invalid accountId',
+      });
+    });
+
+    it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with valid accountId: no msa', async () => {
+      const providerId = provider.msaId?.toString();
+      const keyring = new Keyring({ type: 'sr25519' });
+      const keypair = keyring.addFromUri('//Bob//InvalidMSA');
```

Comment:
```suggestion
      const keypair = new Keyring({ type: 'sr25519' }).createFromUri('//Bob//InvalidMSA');
```

Reply:
THere are a couple of tests that use a valid public key address that is not mapped to an MSA; maybe hoist this to the top of the file in a global `nonMsaAccountAddress` variable?